plugins/manage_engine_service_desk/plugin.spec.yaml

```diff
@@ -0,0 +1,1345 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: manage_engine_service_desk
+title: Manage Engine Service Desk
+description: ManageEngine's Service Desk has the ability to centralize and capture reported issues, this allows security/IT admin to track and manage all incidents in an easy manner. The numerous help desk tickets raised are organized and tracked in the Requests module. The Requests module enables you to handle tickets promptly, assign tickets to technicians, merge similar requests, and so on
+version: 1.0.0
+supported_versions: ["ServiceDesk Plus 13008"]
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/manage_engine_service_desk
```

Comment:
Could you add API-doc link?

Reply:
@jkaluza-rapid7 Done, please have a look